rustup-init.sh

```diff
@@ -80,6 +80,21 @@ main() {
     local _arch="$RETVAL"
     assert_nz "$_arch" "arch"
 
+    local _default_host=""
+    case "$_arch" in
+        *windows*)
+            _default_host="--default-host $_arch"
```

Comment:
I think this is okay, but the variable name here is a bit confusing. Maybe this variable can be renamed to `_default_host_override` to emphasize the fact that it's really an override.